docs/basics/101-128-summary.rst

```diff
@@ -0,0 +1,46 @@
+.. _summary_config:
+
+Summary
+-------
+
+This has been a substantial amount of information on various configuration
+types, methods, and files. After this lecture, you have greatly broadened
+your horizon about configurations of datasets:
+
+- Configurations exist at different scopes and for different tools. Each
+  of such configuration scopes exists in an individual file, on a *system-wide*,
+  *global* (user-specific) or *local* (repository specific) level. More
+  specialized scopes take precedence over more global scopes.
```

Comment:
See above. There's *dataset* level in addition (`.datalad/config`), which is committed and thereby distributed in opposition to *local* (`.git/config`).